apps/labs/posts/jupyter-everywhere.md

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Jupyter Everywhere: empowering interactive computing for K-12 education' # not final
+published: November 01, 2025
+authors: [agriya-khetarpal] # ?[agriya-khetarpal, peyton-murray, michał-krassowski] how to add multiple authors as peyton and mike are at OpenTeams now?
+description: 'The story of how Jupyter Everywhere is transforming K-12 education through interactive computing'
+# "Jupyter" used to be a category for older blogs; it no longer is. "Interactive computing" could
+# subsume it as well, if not "Jupyter". ig "Developer workflows" is not quite right... so for now:
+category: ["OSS Experience"]
+featuredImage:
+  src: /posts/jupyter-everywhere/placeholder.png
+  alt: placeholder
+hero:
+  # JE octopus logo here
+  imageSrc: placeholder
+  imageAlt: placeholder
+---
+
+# The Jupyter Everywhere story: empowering interactive computing for K-12 education
+<!-- the title is tentative and not final -->
+
+Hi! I am Agriya Khetarpal, a software engineer at Quansight PBC. In this blog post, I will share the story of how we built [Jupyter Everywhere](https://jupytereverywhere.org/), an innovative notebooks-based end-to-end application for K-12 education for high school students in the U.S.A., in collaboration with [Skew The Script](https://skewthescript.org/) and [CourseKata](https://coursekata.com/).
+
+Jupyter Everywhere aims to make interactive computing accessible to students and educators, regardless of their technical background or resources. We stand on the shoulders of giants, leveraging the power of [JupyterLite](https://jupyterlite.readthedocs.io/en/latest/) and [WebAssembly (WASM)](https://webassembly.org/) – cutting-edge technologies that enable running Jupyter notebooks entirely in a web browser. In particular, we use [Pyodide](https://pyodide.org/en/stable/), a WASM-based distribution of Python with a rich scientific stack, to execute data science and statistical computing code in a browser without provisioning any server-side dependencies or deployments.
+
+We'll discuss the challenges we faced, the features we implemented, and the lessons we learned while developing Jupyter Everywhere. I hope that this post will inspire educators, developers, and institutions to explore the potential of interactive computing in education, and to contribute to the open source ecosystems that make it all possible and worthwhile.
```

Comment:
If the first paragraph starts with "I", I would keep this one as such too - you are telling the story of how "we" built things. This is especially glaring because second sentence in this paragraph starts with "I" again.

```suggestion
I'll discuss the challenges we faced, the features we implemented, and the lessons we learned while developing Jupyter Everywhere. I hope that this post will inspire educators, developers, and institutions to explore the potential of interactive computing in education, and to contribute to the open source ecosystems that make it all possible and worthwhile.
```